### PR TITLE
Add bundle to yaml

### PR DESCRIPTION
**Pull request by clairetag**

### CODE REVIEW

File: kable-api-v1.yaml [hunk 1]
```diff
@@ -1415,6 +1415,307 @@ paths:
...
+        - in: header
+          name: Kable-Client-Id
+          description: Your client ID, found in the dashboard of your Kable account.
+          schema:
+            type: string
+          example: kci_3c90e9ac92c64f31ae8ed84d21e18740
+          required: true
+        - in: header
+          name: Kable-Client-Secret
+          description: Your `LIVE` or `TEST` secret key. Customers exist across all environments, so it does not matter which environment you use to fetch customers.
```
Comment (adamsommer) [inline kable-api-v1.yaml]: `description: Your `LIVE` or `TEST` secret key.`

(You can drop the bit about customers)

File: kable-api-v1.yaml [hunk 2]
```diff
@@ -1415,6 +1415,307 @@ paths:
...
```
Comment (adamsommer) [inline kable-api-v1.yaml:1471]: same here

File: kable-api-v1.yaml [hunk 3]
```diff
@@ -1415,6 +1415,307 @@ paths:
...
+        - in: header
+          name: Kable-Client-Id
+          description: Your client ID, found in the dashboard of your Kable account.
+          schema:
+            type: string
+          example: kci_3c90e9ac92c64f31ae8ed84d21e18740
+          required: true
+        - in: header
+          name: Kable-Client-Secret
+          description: Your `LIVE` or `TEST` secret key. Customers exist across all environments, so it does not matter which environment you use to create customers. Each customer will have separate keys for `LIVE` and `TEST` environments of your API.
```
Comment (adamsommer) [inline kable-api-v1.yaml:1513]: same here

File: kable-api-v1.yaml [hunk 4]
```diff
@@ -1415,6 +1415,307 @@ paths:
...
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - name
+              properties:
+                name:
+                  type: string
+                  example: "Messages and Connections Bundle"
```
Comment (adamsommer) [inline kable-api-v1.yaml]: let's use `example: "Professional Package"` as the example here -- I think that will resonate more with different customers

File: kable-api-v1.yaml [hunk 5]
```diff
@@ -1415,6 +1415,307 @@ paths:
...
+                  description: "A human-readable name for the bundle, visible on dashboards, invoices, and reports."
+                externalId:
+                  type: string
+                  example: "yourcompany_bundle123"
+                  description: "An identifier for the bundle as defined by your API."
+                description:
+                  type: string
+                  example: "A usage bundle based on the number of messages and connections"
```
Comment (adamsommer) [inline kable-api-v1.yaml]: `description: "The perfect package for businesses starting to scale."`

(let's keep it abstract and widely applicable -- Bundle.name and Bundle.description are marketing material)

File: kable-api-v1.yaml [hunk 6]
```diff
@@ -1415,6 +1415,307 @@ paths:
...
+                  description: "A human-readable description for the bundle, visible on dashboards, invoices, and reports."
+                nickname:
+                  type: string
+                  example: "messages_connections_v1"
```
Comment (adamsommer) [inline kable-api-v1.yaml:1541]: `example: "professional_2022_v2"`

File: kable-api-v1.yaml [hunk 7]
```diff
@@ -1415,6 +1415,307 @@ paths:
...
+                  description: An additional identifier for the bundle, defined by you, that is *not* visible to customers. If you have different *Monthly Active Users Bundle*s for different cohorts of customers, this is a useful place to differentiate those.
```
Comment (adamsommer) [inline kable-api-v1.yaml:1542]: `description: "An additional identifier for the bundle, defined by you, that is *not* visible to customers. If you have multiple *Starter Tier*s for different customers, this is a helpful way to differentiate."`

File: kable-api-v1.yaml [hunk 8]
```diff
@@ -1415,6 +1415,307 @@ paths:
...
+                planIds:
+                  type: array
+                  items:
+                    type: string
+                  description: "A collection of plans to include in the bundle."
+                  example: ["pln_a23e148584d0463482c961b7f62f824c"]
+                costPerCredit:
+                  type: number
+                  example: 1.2
```
Comment (adamsommer) [inline kable-api-v1.yaml]: let's make this look more like money -- `0.25` rather than one decimal place

File: kable-api-v1.yaml [hunk 9]
```diff
@@ -1415,6 +1415,307 @@ paths:
...
+                  description: An additional identifier for the bundle, defined by you, that is *not* visible to customers. If you have different *Monthly Active Users Bundle*s for different cohorts of customers, this is a useful place to differentiate those.
```
Comment (adamsommer) [inline kable-api-v1.yaml]: `description: "Overage costs per credit for bundle subscribers (only relevant if bundle plans are priced in credits).`

File: kable-api-v1.yaml (hunk 1 above)
Comment (adamsommer) [reply kable-api-v1.yaml]: This comment applies throughout this PR -- same edit to each method

File: kable-api-v1.yaml [hunk 10]
```diff
@@ -1415,6 +1415,307 @@ paths:
...
+          schema:
+            type: string
+          example: sk_test.jI92Cbu0.XeHWdYM1VTLy4oLtGMw8wrmpt5q9d04n
+          required: true
+        - in: path
+          name: bundleId
+          description: The identifier for the bundle. You can pass in *either* the `bundleId` (as defined by Kable) or the `externalId` (as defined by you).
+          schema:
+            type: string
+          example: pln_a23e148584d0463482c961b7f62f824c
```
Comment (adamsommer) [inline kable-api-v1.yaml]: let's use a `bun_...` example  here, not a Plan ID

File: kable-api-v1.yaml [hunk 11]
```diff
@@ -1415,6 +1415,307 @@ paths:
...
+          required: true
+      requestBody:
+        description: Information about the bundle to create.
```
Comment (adamsommer) [inline kable-api-v1.yaml]: "update" not "create"

**Review by adamsommer — CHANGES_REQUESTED** [review]
@clairetag this looks great overall. Some comments below that actually appear a few times in in a few locations

My general thoughts behind these comments are basically driven by "how can we make this resonate with a wider cohort of readers"? The more widely-applicable an example is, the better. (Also a few spots where copy-paste needs some cleanup)

Ping me again for a final pass and we can deploy this tomorrow 👍

File: kable-api-v1.yaml (hunk 5 above)
Comment (clairetag) [reply kable-api-v1.yaml]: great point on the marketing comment! I agree

**Review by adamsommer — APPROVED** [review]

**Pull request merged**